annotations/views.py

```diff
@@ -268,11 +268,16 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
      "aff":"affiliation", "loc":"location"}
 
+    srch_term = request.GET.get('srch-term')
```

Comment:
No need to neglect vowels, ``search_term`` and ``search-term`` are much clearer and don't make me feel like my brain is getting squeezed in a vice ;-)